typescript/framework-extensions/langchain/README.md

```diff
@@ -13,7 +13,7 @@ LangChain extension of AgentKit. Enables agentic workflows to interact with onch
 ### Installation
 
 ```bash
-npm install @coinbase/agentkit-langchain @coinbase/agentkit @langchain @langchain/langgraph @langchain/openai
+pnpm add @coinbase/agentkit-langchain @coinbase/agentkit @langchain @langchain/langgraph @langchain/openai
```

Comment:
`pnpm install` only installs from package.json, and cannot be used to add modules. `pnpm add` must be used instead to introduce new dependencies

Reply:
This has been reverted, as we no longer recommend users to use pnpm. This refactor is only relevant for contributors